analysis/parameter_tuning.py

```diff
@@ -153,6 +163,31 @@ def _find_candidates(histogram: histograms.Histogram) -> List:
         max_contributions_per_partition=linf_bounds)
 
 
+def _find_candidates_quantiles(histogram: histograms.Histogram) -> List:
+    QUANTILES_TO_USE = [0.9, 0.95, 0.98, 0.99, 0.995]
+    candidates = histogram.quantiles(QUANTILES_TO_USE)
+    candidates.append(histogram.max_value)
+    candidates = list(set(candidates))  # remove duplicates
+    candidates.sort()
+    return candidates
+
+
+def _find_candidates_constant_relative_step_func(n_max: int):
+
+    def _find_candidates_constant_relative_step(
```

Comment:
Nit: it would be nice to simplify code by removing returning function. It looks it can be done by introducing in `_find_candidates_quantiles` function the argument `n_max` . WDYT?